ui/raidboss/data/02-arr/dungeon/haukke_manor.js

```diff
@@ -7,8 +7,8 @@
       // Dark Mist Stun with Suppression for duplicate messages
       id: 'Haukke Normal Dark Mist Stun',
       netRegex: NetRegexes.startsUsing({ id: '2C1', source: ['Manor Maidservant', 'Manor Claviger', 'Lady Amandine'] }),
-      condition: function(data) {
-        return data.canStun();
+      condition: function(data, matches) {
+        return data.CanStun();
       },
       suppressSeconds: 2,
       response: Responses.stun('info'),
```

Comment:
Note that here and elsewhere, the condition doesn't require `matches`, just `data`. An individual player's ability to stun is not dependent on the results of the regex match.

(That being said, `responses.stun()` *does* require the captured match results, so this particular trigger needs to capture, which you have it correctly set up to do. If you're ever concerned about whether a `response` helper needs the results of the regex match, you can [look it up](https://github.com/quisquous/cactbot/blob/main/resources/responses.js) and check what its parameters are.)